html/webappapis/dynamic-markup-insertion/opening-the-input-stream/remove-initial-about-blankness.window.js

```diff
@@ -0,0 +1,65 @@
+// This tests the issues discussed in https://github.com/whatwg/html/issues/4299
+// and fixed in https://github.com/whatwg/html/pull/6567.
+
+// Note: because browsers do not interoperate on the spec's notion of window reuse (see e.g. https://crbug.com/778318)
+// we pick a specific interoperable test case, which is "currently on initial about:blank, but loading something".
+
+async_test(t => {
+  const iframe = document.createElement("iframe");
+
+  // We can't just leave it at the actual initial about:blank because of the interop issues mentioned above.
+  // So put it in the "currently on initial about:blank, but loading something" state which interoperably does Window
+  // reuse.
+  iframe.src = "/common/blank.html";
+
+  // Create the Window object. It will be for the initial about:blank since the load of /common/blank.html hasn't
+  // completed.
+  document.body.append(iframe);
```

Comment:
One quibble about these comments, it'll only start loading/navigating right here, right?

Reply:
Yeah, the comment on lines 10-12 applies to both lines 13 and 17.